docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -253,6 +253,51 @@ include::partial$dynamic-variables-ui.adoc[]
 
 == Profiles
 
+Profiles allow to choose or parameterize browser that tests run on. See more details on the profiles and nesting strategy on the xref:ROOT:tests-configuration.adoc#_profiles[configuration page].
+
+Web-app plugin contains significant amount of built-in profiles. They can be found after plugin installation by the following path:
+[source, DIR]
+----
+Referenced Libraries -> vividus-plugin-web-app-version -> properties -> profile -> web
+----
+
+Let's go through the structure and review its content.
+
+=== Desktop
+
+General place for web profiles. Includes the following browsers' directories:
+
+- chrome
```

Comment:
I propose to use table view
|profile|description|
|-|-|
|wed/desktop/chrome|command line arguments set this way bla-bla|
|web/desktop/safari|

It will improve UX, because usually someone just wants to run on some specific browser, and it is better to just have a value to put into configuration